### ISSUE TITLE
Wrong physics when accelerating and braking using vehicle wheel

### ISSUE THREAD

**Issue by JustElectron** [godotengine/godot#34856]
The spring physics of the vehicle wheel or body is wrong. When accelerating and braking, the vehicle tilts forward or backward. However this is done in the opposite direction from what is natural.
Using a simple setup with a vehicle body and 4 wheels and a static ground plane.

- When applying an engine force to the vehicle, the vehicle tilts forward. In real life the vehicle would tilt backwards.

- When braking the vehicle tilts backwards. In real life the vehicle would tilt forward.

Here is an example of a vehicle braking
![Capture](https://user-images.githubusercontent.com/48671787/71813295-8b08b080-3079-11ea-915f-b25609b1ebf5.PNG)

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in Godot 3.2.3 or any later release?

If yes, please ensure that an up-to-date Minimal Reproduction Project (MRP) is included in this report (a MRP is a zipped Godot project with the minimal elements necessary to reliably trigger the bug). You can upload ZIP files in an issue comment with a drag and drop.

**Comment by angad-k** [comment]
Couldn't reproduce the issue in 3.2.3. Tilting works well. Recording : 

https://user-images.githubusercontent.com/54415525/110321977-7090e500-8038-11eb-8943-bf920f822dd5.mp4

**Comment by KoBeWi** [comment]
So closing as resolved. If you can still reproduce in Godot 3.2.4 rc3 or never, comment with a reproduction project.

**Comment by IvailoBurov** [comment]
For me, this problem exists in all versions, including 3.3.3. I'm attaching a video. The BastiaanOlij project can be used for a test project, where the horizon changes downwards during acceleration: https://github.com/BastiaanOlij/vehicle-demo/tree/Part-4

https://user-images.githubusercontent.com/63282037/133959509-45662d06-ec57-4566-b424-3800c5135dbe.mp4

If necessary, I could do a separate project, but I use Bastian's code here.

**Comment by KoBeWi** [comment]
Reopening as per above comment.

**Comment by mojoyup** [comment]
Same for me as of now. Version 3.4.4 Steam version.

**Comment by Calinou** [comment]
@mojoyup Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by mojoyup** [comment]
> @mojoyup Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

Interestingly, I created a simplified test project, same version and it seems to work. 
I will try to find out in my original what could have caused this and report back, in case anyone has this issue also.

 What happened with mine was, when I was changing the VehicleWheel Radius (I scaled everything down, even the models/meshes), then it seems to be "backwards". 

The project before scaling and resizing the VehicleWheels worked but Ill try to get more details for it if needed.

**Comment by mojoyup** [comment]
[VPTest.zip](https://github.com/godotengine/godot/files/8951978/VPTest.zip)
This is the test file. This works if anyone wants to use this

**Comment by Zireael07** [comment]
Rule of thumb for physics bodies/collision shapes is NOT to scale them!

**Comment by mojoyup** [comment]
The VehicleWheels have the option for such in the parameters. The others were not scaled in the editor, but in Sketchup.

**Comment by mojoyup** [comment]
I think I might be able to ahve this closed. I figured out one thing that affects the physics this way. It is the different in the transforms of the parent node (Vehicle), and the children nodes (most likely and mainly the wheels and main collision). I have always put my origin of the children nodes to where the parent node sits on the ground for better control. If it is too exotic of a difference between the parent node Y pos and the children Y pos', Itll start to behave different. I hope that makes sense. 

TL/DR: Try to reset the transforms of the children and the parent node(Vehicle) and test/tweak
